spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/RawModeKafkaBinderTests.java

```diff
@@ -288,15 +218,16 @@ public void testSendAndReceivePubSub() throws Exception {
 		QueueChannel module2InputChannel = new QueueChannel();
 		QueueChannel module3InputChannel = new QueueChannel();
 		binder.bindProducer("baz.0", moduleOutputChannel, null);
-		binder.bindConsumer("baz.0", moduleInputChannel, null);
+		binder.bindConsumer("baz.0", "test", moduleInputChannel, null);
 		moduleOutputChannel.addInterceptor(new WireTap(tapChannel));
```

Comment:
This needs to be removed, we don't do wiretapping for taps anymore. Also, this causes duplicate messages to be resent to the main topic.